src/bootstrap/src/utils/exec.rs

```diff
@@ -145,28 +142,35 @@ impl CommandProfiler {
 
         entries.sort_by(|a, b| b.2.cmp(&a.2));
 
-        for (key, profile, max_duration) in entries {
+        let mut total_fingerprints = 0;
+        let mut total_cache_hits = 0;
+        let mut total_execution_duration = Duration::ZERO;
+        let mut total_saved_duration = Duration::ZERO;
+
+        for (key, profile, max_duration) in &entries {
             writeln!(writer, "Command: {:?}", key.format_short_cmd()).unwrap();
 
             let mut hits = 0;
             let mut runs = 0;
+            let mut command_total_duration = Duration::ZERO;
 
             for trace in &profile.traces {
                 match trace {
-                    ExecutionTrace::CacheHit { timestamp } => {
+                    ExecutionTrace::CacheHit => {
                         hits += 1;
-                        let time = timestamp.duration_since(UNIX_EPOCH).unwrap();
-                        writeln!(writer, "  - Cache hit at:   {time:?}").unwrap();
                     }
-                    ExecutionTrace::Executed { duration, timestamp } => {
+                    ExecutionTrace::Executed { duration, .. } => {
                         runs += 1;
-                        let time = timestamp.duration_since(UNIX_EPOCH).unwrap();
-                        writeln!(writer, "  - Executed at:    {time:?}, duration: {duration:.2?}")
-                            .unwrap();
+                        command_total_duration += *duration;
                     }
                 }
             }
 
+            total_fingerprints += 1;
+            total_cache_hits += hits;
+            total_execution_duration += command_total_duration;
+            total_saved_duration += command_total_duration * hits as u32;
```

Comment:
This only works because we either get exactly 1 execution (and the rest are cache hits) or N executions (and there are zero cache hits). Could you please add a comment that explains this? Otherwise the computation is weird (because we multiply total command duration by the number of cache hits).